amazon-kinesis-client/src/main/java/software/amazon/kinesis/leases/dynamodb/DynamoDBLeaseCoordinator.java

```diff
@@ -100,6 +110,8 @@ public DynamoDBLeaseCoordinator(final LeaseRefresher leaseRefresher,
                                     final int maxLeasesForWorker,
                                     final int maxLeasesToStealAtOneTime,
                                     final int maxLeaseRenewerThreadCount,
+                                    final long initialLeaseTableReadCapacity,
```

Comment:
As we just added the ` @KinesisClientInternalApi` retain the existing constructor and chain them together.  You can mark the original constructor as deprecated.